### ISSUE TITLE
iOS Have token, but don't receive notifications.

### ISSUE THREAD

**Issue by luisfuertes** [evollu/react-native-fcm#144]
Recently i change the Bundle id of my app and the google firebase account for prepare App to production. I update GoogleService-Info.plist and upload in firebase console the push certificate.

Library give me the token correctly. But when i try a send notification from firebase console, the notification never receive.

What can i check to do it works?

Thanks!

**Comment by evollu** [comment]
try sending through API and see if any error is returned.

**Comment by luisfuertes** [comment]
How? Local notification?

**Comment by evollu** [comment]
i mean post to https://fcm.googleapis.com/fcm/send

**Comment by luisfuertes** [comment]
Im trying from Firebase console, and error: 'Not yet registered the registration token'

![captura de pantalla 2016-10-10 a las 10 19 31](https://cloud.githubusercontent.com/assets/18212615/19230206/891c30b2-8ed3-11e6-89a8-805a8c4a4291.png)

**Comment by evollu** [comment]
that is a long EDIT 😄 
It is expected that you will receive the notification twice majorly because someone may want to update the app in background. You can distinguish them by check `opened_from_tray` flag.

Looks like you've solved couple problems yourself. Can you summarize what is still outstanding now?

**Comment by luisfuertes** [comment]
@evollu  thx for answer. I have all working, but my notif object dont have the prop "opened_from_tray" and "local_notification", only receive a oject with a type.

How can i fix it?

**Comment by evollu** [comment]
if you don't see the property usually it means false

**Comment by sibelius** [comment]
@luisfuertes can u close this?
